src/layer/__tests__/cache.ts

```diff
@@ -113,3 +113,188 @@ describe('Testing caching', () => {
     expect(mockNetwork.history.post.length).toBe(2);
   });
 });
+
+describe('Testing cache targets', () => {
+  beforeEach(() => {
+    Object.assign(global, makeServiceWorkerEnv(), fetch);
+    jest.resetModules();
+    memoryCache.clear();
+  });
+
+  it('should cache to cache api', async () => {
+    const { fromTime, toTime, bbox, layer, mockedResponse, expectedResultTiles } = constructFixtureFindTiles(
+      {},
+    );
+    const requestsConfig = {
+      cache: {
+        expiresIn: 60,
+        targets: [CacheTarget.CACHE_API],
+      },
+    };
+    mockNetwork.reset();
+    mockNetwork.onPost().replyOnce(200, mockedResponse);
+    mockNetwork.onPost().replyOnce(200, mockedResponse);
+
+    const responseFromMockNetwork = await layer.findTiles(bbox, fromTime, toTime, null, null, requestsConfig);
+    const fromCacheResponse = await layer.findTiles(bbox, fromTime, toTime, null, null, requestsConfig);
+
```

Comment:
I would suggest adding another assert here:
```
  expect(mockNetwork.history.post.length).toBe(1);
```

Actually, I think the tests would be much stricter (and more readable) if every `findTiles` call would be followed with an `expect`, if you agree?

Reply:
I have restructured the tests a bit per your suggestion